docs/contributing.rst

```diff
@@ -106,29 +100,41 @@ Commit changes to git::
 
     git commit -a -m "Prepare release <new-version>"
 
-Test the new version with Tox_::
+Push changes to your fork and open a pull request::
 
-    tox -r
+    git push --set-upstream <fork-name> release-<new-version>
+
+Wait for the tests to pass and the pull request to be approved. Merge the code
+to master::
 
-Create source distribution::
+    git checkout master
+    git merge --ff-only release-<new-version>
+    git push <upstream-name> master <new-version>
```

Comment:
`<new-version>` is only tagged in the next step, you can probably omit it here.

Reply:
done